executor/aggregate_test.go

```diff
@@ -39,53 +39,53 @@ func (s *testSuite) TestAggregation(c *C) {
 	tk.MustQuery("select bit_xor(c) from t where NULL").Check(testkit.Rows("0"))
 	result := tk.MustQuery("select count(*) from t")
 	result.Check(testkit.Rows("7"))
-	result = tk.MustQuery("select count(*) from t group by d")
+	result = tk.MustQuery("select count(*) from t group by d order by c")
 	result.Check(testkit.Rows("3", "2", "2"))
 	result = tk.MustQuery("select distinct 99 from t group by d having d > 0")
 	result.Check(testkit.Rows("99"))
 	result = tk.MustQuery("select count(*) from t having 1 = 0")
 	result.Check(testkit.Rows())
-	result = tk.MustQuery("select c,d from t group by d")
+	result = tk.MustQuery("select c,d from t group by d order by d")
 	result.Check(testkit.Rows("<nil> 1", "1 2", "1 3"))
 	result = tk.MustQuery("select - c, c as d from t group by c having null not between c and avg(distinct d) - d")
 	result.Check(testkit.Rows())
 	result = tk.MustQuery("select - c as c from t group by c having t.c > 5")
 	result.Check(testkit.Rows())
 	result = tk.MustQuery("select t1.c from t t1, t t2 group by c having c > 5")
 	result.Check(testkit.Rows())
-	result = tk.MustQuery("select count(*) from (select d, c from t) k where d != 0 group by d")
+	result = tk.MustQuery("select count(*) from (select d, c from t) k where d != 0 group by d order by c")
```

Comment:
Test.....................................................................Ignore this